.travis.yml

```diff
@@ -16,6 +16,8 @@ before_install:
 script:
   - ./autogen.sh 
   - ./configure --enable-zabbix-${ZBX_HEADERS_VERSION}
+  - clang-format -i --style=file src/*.c
+  - git diff --exit-code
```

Comment:
Any idea why `make` is broken after clang-format applied?
https://travis-ci.org/v-zhuravlev/libzbxmodbus/jobs/426184921#L2224

Reply:
Because of [this](https://travis-ci.org/v-zhuravlev/libzbxmodbus/jobs/426184921#L832-L834), I suppose.

Reply:
Thank you, fixed